src/AVFoundation/Enums.cs

```diff
@@ -836,5 +840,137 @@ public enum AVAssetExportSessionPreset {
 		AppleM4A = 8, // AVAssetExportPresetAppleM4A
 		[Field ("AVAssetExportPresetPassthrough")]
 		Passthrough = 9, // AVAssetExportPresetPassthrough
+
+		[iOS (11, 0), Mac (10, 13)]
+		[Field ("AVOutputSettingsPresetHEVC1920x1080")]
+		PresetHevc1920x1080 = 11, 
+
+		[iOS (11, 0), Mac (10, 13)]
+		[Field ("AVOutputSettingsPresetHEVC3840x2160")]
+		PresetHevc3840x2160 = 12,
+	}
+
+	[TV (11,0), NoWatch, iOS (11,0)]
+	[Native]
+	public enum AVDepthDataAccuracy : nint {
+		Relative = 0,
+		Absolute = 1
```

Comment:
Minor: missing trailing comma after the last element (there are more enums like this below too).